poster/styles/styles.scss

```
@@ -0,0 +1,296 @@
+body {
+    background-color: #FFC9B5;
+}
+
+.background-image {
+    right: 20px;
+    top: 100px;
+    width: 700px;
+    position: fixed;
+}
+
+.background-image-container {
+    position: relative;
+}
+
+.info-secondary {
+    position: fixed;
+    bottom: 80px;
+    right: 500px;
+    color: #FFFFFF;
+    font-family: "brevia", sans-serif;
+    font-size: 12px;
+
+    p {
+        transform: rotate(-35deg);
+        display: inline-block;
+    }
+}
+
+.header {
+    font-size: 25px;
+    color: #4C0020;
+    font-family: "scriptorama-tradeshow-jf", sans-serif;
+    padding-left: 115px;
+    padding-top: 40px;
+    line-height: 0px;
```

Zero measurements do not need units.
```suggestion
    line-height: 0;
```